TeleSharp.Generator/TeleSharp.Generator.csproj

```diff
@@ -9,9 +8,15 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>TeleSharp.Generator</RootNamespace>
     <AssemblyName>TeleSharp.Generator</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
```

Comment:
there's no reason to change this project's targetFramework, projects with .NETFramework 4.x can still link to .NETStandard libraries